worker/orca_grader/container/do_grading.py

```diff
@@ -2,7 +2,6 @@
 import os
 import sys
 import shutil
```

Comment:
I don't think we need shutil; I added it for the unsuccessful debugging attempts